.github/workflows/ci.yml

```diff
@@ -79,24 +79,20 @@ jobs:
     name: Windows, OTP-${{ matrix.otp_release }}, Windows Server 2019
     strategy:
       matrix:
-        otp_release: ['24.0']
+        otp_release: ['24']
```

Comment:
Should we need to add OTP 25 to the pipeline?

Reply:
If we need to set the same like test linux does. So it should be 24.0, 24.3 and 25

Reply:
Let's do 24 and 25, we don't need to be as through.

Reply:
Fixed. 🙇